pymilvus/orm/collection.py

```diff
@@ -798,7 +798,7 @@ def search_iterator(
         data: List,
         anns_field: str,
         param: Dict,
-        limit: int,
+        batch_size: Optional[int] = 1000,
```

Comment:
Why adding defaults? Comparing to the no defaults limit.

Reply:
https://github.com/milvus-io/milvus/issues/26358#issuecomment-1678500383

Reply:
required by clients, in case they don't provide batch_size